account_invoice_report_credentials/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Author: Silvija Butko. Copyright: JSC Focusate.
+# See LICENSE file for full copyright and licensing details.
+{
+    'name': "Account Invoice Report Credentials",
+    'version': '14.0.2.0.0-rc.1',
+    'license': 'LGPL-3',
```

Comment:
Aren't we using `OEEL-1`?
```suggestion
    'license': 'OEEL-1',
```

Reply:
No, for open source modules, it does not make sense to use it as it stands for `Odoo Enterprise Edition License v1.0`. Odoo community also uses `LGPL-3`.

Reply:
Yeah, comment was before I've realized modules are going to be public.